config/always-python.m4

```diff
@@ -12,6 +12,17 @@ AC_DEFUN([ZFS_AC_PYTHON_VERSION], [
 	])
 ])
 
+dnl #
+dnl # ZFS_AC_PYTHON_VERSION_IS_2
+dnl # ZFS_AC_PYTHON_VERSION_IS_3
+dnl #
+dnl # Tests if the $PYTHON_VERSION matches 2.x or 3.x.
+dnl #
+AC_DEFUN([ZFS_AC_PYTHON_VERSION_IS_2],
+	[expr "${PYTHON_VERSION}" : 2\\. >/dev/null])
```

Comment:
It's possible to avoid external expr commands:

`test "${PYTHON_VERSION%"${PYTHON_VERSION#??}"}" = "2."`